enginetest/queries/script_queries.go

```diff
@@ -2013,6 +2013,21 @@ var ScriptTests = []ScriptTest{
 			},
 		},
 	},
+	{
+		Name: "Describe with expression correctly works",
+		SetUpScript: []string{
+			"CREATE TABLE t(pk int primary key, val int DEFAULT (pk * 2))",
+		},
+		Assertions: []ScriptTestAssertion{
+			{
+				Query: "DESCRIBE t",
+				Expected: []sql.Row{
+					{"pk", "int", "NO", "PRI", "NULL", ""},
+					{"val", "int", "YES", "", "((pk * 2))", ""},
```

Comment:
This doesn't match the MySQL behavior (extra parens on  the default expr)

Reply:
MySQL itself is not consistent about this. Just file an issue for now, we'll want to fix it at some point.

```
mysql> desc t;
+-------+------+------+-----+------------+-------------------+
| Field | Type | Null | Key | Default    | Extra             |
+-------+------+------+-----+------------+-------------------+
| pk    | int  | NO   | PRI | NULL       |                   |
| val   | int  | YES  |     | (`pk` * 2) | DEFAULT_GENERATED |
+-------+------+------+-----+------------+-------------------+
2 rows in set (0.02 sec)

mysql> show create table t;
+-------+-------------------------------------------------------------------------------------------------------------------------------------------------------------------+
| Table | Create Table
                                                    |
+-------+-------------------------------------------------------------------------------------------------------------------------------------------------------------------+
| t     | CREATE TABLE `t` (
  `pk` int NOT NULL,
  `val` int DEFAULT ((`pk` * 2)),
  PRIMARY KEY (`pk`)
) ENGINE=InnoDB DEFAULT CHARSET=utf8mb4 COLLATE=utf8mb4_0900_ai_ci |
+-------+-------------------------------------------------------------------------------------------------------------------------------------------------------------------+
```

Reply:
Sounds good